src/main/java/com/epam/aidial/core/data/Publication.java

```diff
@@ -19,19 +19,20 @@ public class Publication {
     /**
      * Target directory url without resource prefix to publish to: public/ or public/folder/.
      */
-    String targetUrl;
+    String targetFolder;
```

Comment:
I may confuse because a folder is just a singe name. However the description states that's an URL

Reply:
I changed description